src/content/en/2022/css.md

```diff
@@ -6,17 +6,1321 @@ description: CSS chapter of the 2022 Web Almanac covering trends, changes, and p
 authors: [rachelandrew]
 reviewers: [j9t, meyerweb, svgeesus]
 analysts: [rviscomi]
-editors: [dereknahman]
+editors: [rviscomi]
 translators: []
+rachelandrew_bio: TODO
 results: https://docs.google.com/spreadsheets/d/1OU8ahxC5oYU8VRryQs9BzHToaXcOntVlh6KUHjm15G4/
-featured_quote: TODO
-featured_stat_1: TODO
-featured_stat_label_1: TODO
-featured_stat_2: TODO
-featured_stat_label_2: TODO
-featured_stat_3: TODO
-featured_stat_label_3: TODO
-unedited: true
+featured_quote: The past few years have seen a flurry of new CSS features. Many of these have taken inspiration from things developers were already doing with JavaScript or in preprocessors, while others provide methods of doing things that were impossible a few years ago. Having new features available is one thing, but are developers actually using them in their production web pages and applications?
```

Comment:
@rachelandrew I've added some featured quote/stats that I think are interesting but feel free to suggest something else.